### PR TITLE
Feature 67 event card design

### PR DESCRIPTION
**Pull request by Martijn0405**
### What is the change?

Added an event component as described in the issue description #67 using the Figma design

### Before / After Change Screenshots

image2

### CODE REVIEW

**Comment by harshith-venkatesh** [comment]
@Martijn0405 , We would be updating the template where we have mandated to add unit test cases for the PR that would be raised since we are observing event card components that could be used in multiple areas of the product.
We need to ensure it is properly tested.
Please look into that.
Good work on building this component

File: tailwind.config.js [hunk 1]
```diff
@@ -4,12 +4,33 @@ module.exports = {
...
+          DEFAULT: '#D6D3D1',
+          med: '#78716C',
+          dark: '#44403C',
+        },
         resolutionBlue: '#041484',
         razzmatazz: '#E30464',
         feijoa: '#96D882',
         fern: '#6DB470',
       },
+      fontSize: {
```
Comment (isVivek99) [inline tailwind.config.js:21]: we dont need to be updating tailwind config in this `PR`, also i think we have the option to already add font size as 14px, 18px by default from tailwind

**Comment by Martijn0405** [comment]
@harshith-venkatesh I added the test cases. @isVivek99 I reverted the changes done to tailwind.config file.

**Comment by harshith-venkatesh** [comment]
@Martijn0405 , sorry for the delay.
The test cases are failing due to addition of styles which are not been used or created
Screenshot 2023-05-29 at 3 58 15 PM


Could you please resolve this from your end and commit the changes.
I tried from my end as it has been delayed.
However, it is creating the issue

**Comment by harshith-venkatesh** [comment]
Closing the PR, thanks for the contribution

**Pull request closed without merging**